jsonschema-generator/src/main/java/com/github/victools/jsonschema/generator/Option.java

```diff
@@ -87,6 +88,12 @@ public enum Option {
      * @see Option#SIMPLIFIED_OPTIONALS
      */
     FLATTENED_OPTIONALS(FlattenedOptionalModule::new, null, Option.SIMPLIFIED_OPTIONALS),
+    /**
+     * Wheter a {@link java.util.function.Supplier Supplier&lt;T&gt;} instance should be treated as nullable value of the wrapped type {@code T}.
```

Comment:
To me, the `Supplier<T>` does not necessarily convey that it is "nullable", like the `Optional<T>` does.
The `Supplier<T>` is more to enable lazy-loading of particular values, isn't it? Therefore I'd omit the "nullable" part actually.

Reply:
I barely know Java myself I don't quite understand what a `Supplier<T>` is for, what I found though is that the supplied T might be a nullable Type like `Long`.  So it probably makes sense to have the schema to reflect the property can be `anyOf` of `T` or `null`.

Reply:
In the module for the `Optional`, the wrapped type is explicitly set to be null-able even if the default behavior is to treat everything else as not null-able.
I merely prefer to let the `Supplier` not enforce it one way or another and rather let the otherwise configured default do its thing. 😉